kaya_vision_manager:
  ros__parameters:
    pipeline_components: ["kaya_vision://kaya_vision::KayaVisionChatter", "kaya_vision://kaya_vision::KayaRealSenseCamera", "kaya_vision://kaya_vision::KayaTFExtinsics"]
kaya_vision_chatter:
  ros__parameters:
    message: "hello from vision chatter"
kaya_realsense_camera:
  ros__parameters:
    serial_no: "819312073065"
    preset: "package://kaya_vision/config/HighResHighAccuracyPreset.json"
    base_frame_id: camera_link
    extrinsics_rate_hz: 1.0
    aligned:
      enabled: false
      resolution: [848,480]
      fps: 60
    color:
      enabled: true
      resolution: [848,480]
      fps: 60
      rs_stream: "RS2_STREAM_COLOR"
      rs_encoding: "RS2_FORMAT_RGB8"
      image_encoding: "rgb8"
      image_topic: "/camera/color/image_raw"
      info_topic: "camera/color/camera_info"
      frame: "camera_color_optical_frame"
      extrinsics_reference: "infra1"
    depth:
      enabled: true
      resolution: [848,480]
      fps: 60
      rs_stream: "RS2_STREAM_DEPTH"
      rs_encoding: "RS2_FORMAT_Z16"
      image_encoding: "16UC1"
      image_topic: "/camera/depth/image_raw_rect"
      info_topic: "camera/depth/camera_info"
      frame: "camera_depth_optical_frame"
      extrinsics_reference: "infra1"
    infra1:
      enabled: true
      resolution: [848,480]
      fps: 60
      rs_stream: "RS2_STREAM_INFRARED@1"
      rs_encoding: "RS2_FORMAT_Y8"
      image_encoding: "8UC1"
      image_topic: "/camera/infra1/image_raw_rect"
      info_topic: "camera/infra1/camera_info"
      frame: "camera_infra1_optical_frame"
      extrinsics_reference: "infra1"
    infra2:
      enabled: false
      resolution: [848,480]
      fps: 60
      rs_stream: "RS2_STREAM_INFRARED@2"
      rs_encoding: "RS2_FORMAT_Y8"
      image_encoding: "8UC1"
      image_topic: "/camera/infra2/image_raw_rect"
      info_topic: "camera/infra2/camera_info"
      frame: "camera_infra2_optical_frame"
      extrinsics_reference: "infra1"
    accel:
      enabled: false
    gyro:
      enabled: false
kaya_tf_extrinsics:
  ros__parameters:
    cameras: ["/camera/color", "/camera/depth", "/camera/infra1"]
    